pages/requests/[uuid].js

```diff
@@ -55,9 +58,9 @@ const Request = () => {
         setIsPOError(isPOError)
       })()
     }
-  }, [allPOs, isPOError, request, uuid, session, accessToken])
+  }, [request, isLoadingPOs, accessToken, uuid])
 
-  const isLoading = isLoadingRequest || isLoadingSOWs || isLoadingFiles || isLoadingMessages || isLoadingPOs
```

Comment:
not checking for `isLoadingPOs` here allows us to close the sow modal and show the request. if this remained, closing the modal would hide the entire page behind a "loading" spinner.